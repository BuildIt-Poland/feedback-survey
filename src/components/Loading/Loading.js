import React from 'react';
import { Box, Heading } from 'grommet';

const Loading = () => (
  <Box animation="pulse" align="center" justify="center" pad="medium" height="90vh">
    <Heading level="4">Loading...</Heading>
  </Box>
);

export default Loading;
