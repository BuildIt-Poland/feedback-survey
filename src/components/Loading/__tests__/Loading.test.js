import React from 'react';
import { create } from 'react-test-renderer';

import Loading from '../Loading';

describe('COMPONENT - Loading', () => {
  it('renders correctly', () => {
    const component = create(<Loading />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
