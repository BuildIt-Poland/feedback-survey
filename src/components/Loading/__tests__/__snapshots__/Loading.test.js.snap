// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Loading renders correctly 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 hUosMD"
>
  <h3
    className="sc-bdVaJa hFrVFs StyledHeading-sc-1rdh4aw-0 gRvHvF"
  >
    Loading...
  </h3>
</div>
`;
